wxt_cidrbot/cmd_list.py

```diff
@@ -474,25 +474,34 @@ def issues(self, target_user, room_id):
             repo_name = issue.split(', ', 1)[0]
             issue_name = issue_dict[issue]['name']
             status = issue_dict[issue]['assigned_status']
-            if status:
-                assignee = issue_dict[issue]['assigned'].split(", ")
-                for name in assignee:
-                    if name.lower() == git_name_target.lower():
-                        edit_message += f"- Issue located: {issue_name} \n"
-                        if issues_found < 8:
-                            self.webex.edit_message(self.msg_id, edit_message, self.room_of_msg)
-                        url = issue_dict[issue]['url']
-                        issue_type = issue_dict[issue]['type']
-                        issue_num = issue_dict[issue]['number']
-                        issue_color_code = issue_dict[issue]['color_code']
-
-                        issue_type += " #" + str(issue_num)
-                        name_format = issue_name
-                        hyperlink_format = f'<a href="{url}">{name_format}</a>'
-                        text = f"{issue_color_code} &nbsp; {issue_type} in {repo_name}: {hyperlink_format}"
-
-                        message += text + "\n"
-                        issues_found += 1
+
+            if not status:
+                continue
+
+            assignee = issue_dict[issue]['assigned'].split(", ")
+            for name in assignee:
+                if name.lower() == git_name_target.lower():
+                    edit_message += f"- Issue located: {issue_name} \n"
+                    if issues_found < 8:
+                        self.webex.edit_message(self.msg_id, edit_message, self.room_of_msg)
+                    url = issue_dict[issue]['url']
+                    issue_type = issue_dict[issue]['type']
+                    issue_num = issue_dict[issue]['number']
+                    issue_color_code = issue_dict[issue]['color_code']
+                    is_draft = issue_dict[issue]['is_draft']
+
+                    if is_draft:
+                        draft_txt = "(Draft) "
+                    else:
```

Comment:
This doesn't need an else block. Define draft_txt before line 493, and if is_draft, set draft_txt to the appropriate value